### DISCUSSION TITLE
Bug: File name too long

### DISCUSSION THREAD

**Discussion by RobinBially** [open-webui/open-webui#5196] [General]
In logs when chatting with any function, for example Anthropic V2:

`An error occurred: [Errno 63] File name too long: '"""\ntitle: Anthropic Manifold Pipe v2\ndescription: An enhanced implementation of the Anthropic API integration for Open WebUI,\n   [...]  `

**Comment by tjbck** [comment]
Cannot reproduce, more details wanted here.

**Comment by RobinBially** [comment]
Problem is that you call the following function with the function content but expect the file path:

```
 # Extract the frontmatter from the content, simulate file-like behaviour
 frontmatter = extract_frontmatter(
        content
)  # This function needs to handle string inputs
```

```
def extract_frontmatter(file_path):
```

**Reply by tjbck** [reply]
Should be fixed on dev!
